config/routes.php

```diff
@@ -22,13 +22,13 @@
     $map->addRoute(['GET', 'POST'], '/album[/]', 'modules/album/index.php');                         // Photo Album
     $map->addRoute(['GET', 'POST'], '/downloads[/]', 'modules/downloads/index.php');                 // Downloads
     $map->addRoute(['GET', 'POST'], '/forum[/]', 'modules/forum/index.php');                         // Forum
-    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}[/[{id:\d+}[/]]]]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
+    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
     $map->addRoute(['GET', 'POST'], '/help[/]', 'modules/help/index.php');                           // Help
     $map->addRoute(['GET', 'POST'], '/library[/]', 'modules/library/index.php');                     // Articles Library
     $map->addRoute(['GET', 'POST'], '/language[/]', 'modules/language/index.php');                   // Language switcher
     $map->addRoute(['GET', 'POST'], '/login[/]', 'modules/login/index.php');                         // Login / Logout
     $map->addRoute(['GET', 'POST'], '/mail[/]', 'modules/mail/index.php');                           // Personal Messages
-    $map->addRoute(['GET', 'POST'], '/news[/[{action}[/[{id:\d+}[/]]]]]', 'modules/news/index.php'); // News
+    $map->addRoute(['GET', 'POST'], '/news[/[{action}]]', 'modules/news/index.php'); // News
```

Comment:
What's this for???
This change breaks news.

Reply:
oh, I have changed the links in the `news/templates` and it will operate normally with url `domain/modules/act?id=`

Reply:
What is the reason for these changes?

Reply:
I have consulted some people to make this decision. The change also makes the router simpler and more concise later 😊

Reply:
For simplicity you can make route rule as: `$map->addRoute(['GET', 'POST'], '/news/[{action}/[{id:\d+}]]', 'modules/news/index.php');` and replace urls in templates for this rule.

Reply:
In my opinion, the `id` parameter does not represent a file. Therefore it is not so important on URL. If you still want that, I will do it.

Reply:
OK. In our case, this is not critical, but in general, if you make a friendly URL, it is better to do it completely.